test/other/codesize/test_codesize_hello_dylink.gzsize

```diff
@@ -1 +1 @@
-5907
+5926
```

Comment:
Do you know why the codesize for this test changed?

I would imagine that only tests that explicitly use the rpath would see a code size change.

Reply:
I guess it is because I updated libdylink.js?

Reply:
Ah yes, that makes sense.